proposals/20210526-model_load_test_component.md

```diff
@@ -0,0 +1,63 @@
+#### SIG TFX-Addons
+# Project Proposal
+
+**Your name:** Michal Brys
+
+**Your email:** michal.brys@openx.com
+
+**Your company/organization:** OpenX
+
+**Project name:** Model load test TFX Component
+
+## Project Description
+A TFX component performs a load test of the exported TensorFlow model because the prediction time may vary on the model type and structure.
```

Comment:
I just got confused on what's the difference between performance test and load testing, and read https://stackify.com/load-testing-vs-performance-testing-vs-stress-testing/. Can we say that our desired goal is "load testing" not "performance testing"? (because it feels like inference time distribution belong to performance test scope, and thought maybe this component is a PerformanceEvaluator?)